.gitpod.yml

```diff
@@ -19,26 +19,45 @@ tasks:
       echo "Installing dotnet done."
 
     command: |
-      # Check if environment variables are set in https://gitpod.io/variables.
-      echo "Checking if environment variables are set ..."
-      if [ -z ${ADYEN_HMAC_KEY+x} ] || [[ -z ${ADYEN_API_KEY+x} ]] || [[ -z ${ADYEN_CLIENT_KEY+x} ]] || [[ -z ${ADYEN_MERCHANT_ACCOUNT+x} ]]; then
-        echo "Expected environment variables not found. Please set the ADYEN_HMAC_KEY, ADYEN_API_KEY, ADYEN_CLIENT_KEY, ADYEN_MERCHANT_ACCOUNT environment variables and rerun session https://gitpod.io/variables."
-        exit 1
-      fi
-
-      # Run the application based on specified $path.
-      echo "Starting application for '$path' ..."
+      # Check if environment variables are set in https://gitpod.io/variables and run the application based on specified $path.
       case "$path" in
         "checkout-example")
+          if [ -z ${ADYEN_HMAC_KEY+x} ] || [[ -z ${ADYEN_API_KEY+x} ]] || [[ -z ${ADYEN_CLIENT_KEY+x} ]] || [[ -z ${ADYEN_MERCHANT_ACCOUNT+x} ]]; then
+            echo "Expected environment variables not found. Please set the ADYEN_HMAC_KEY, ADYEN_API_KEY, ADYEN_CLIENT_KEY, ADYEN_MERCHANT_ACCOUNT environment variables in https://gitpod.io/variables and restart."
+            exit 1
+          fi
           dotnet run --project checkout-example
+        ;;     
+        
+        "giftcard-example")
+          if [ -z ${ADYEN_HMAC_KEY+x} ] || [[ -z ${ADYEN_API_KEY+x} ]] || [[ -z ${ADYEN_CLIENT_KEY+x} ]] || [[ -z ${ADYEN_MERCHANT_ACCOUNT+x} ]]; then
+            echo "Expected environment variables not found. Please set the ADYEN_HMAC_KEY, ADYEN_API_KEY, ADYEN_CLIENT_KEY, ADYEN_MERCHANT_ACCOUNT environment variables in https://gitpod.io/variables and restart."
+            exit 1
+          fi
+          dotnet run --project giftcard-example
+        ;;
+
+        "paybylink-example")
+          if [ -z ${ADYEN_HMAC_KEY+x} ] || [[ -z ${ADYEN_API_KEY+x} ]] || [[ -z ${ADYEN_MERCHANT_ACCOUNT+x} ]]; then
```

Comment:
Pay By Link doesn't require the `CLIENT_KEY` hence why I had to modify the gitpod file